scout/server/blueprints/variants/views.py

```diff
@@ -208,18 +235,24 @@ def str_variants(institute_id, case_name):
             ("position", pymongo.ASCENDING),
         ]
     )
-    result_size = store.count_variants(case_obj["_id"], {}, None, category)
+
+    result_size = store.count_variants(case_obj["_id"], query, None, category)
+
     data = controllers.str_variants(
-        store, institute_obj, case_obj, variants_query, result_size, page
+        store, institute_obj, case_obj, variants_query, result_size, page=page
```

Comment:
This change is a bit weird, i'd remove =page

Reply:
It is named, but sure, it is also in the right order so will work anyway. It is presumably a leftover from testing something else.. 😄